### ISSUE TITLE
iOS 15 WKWebview rendering blank

### ISSUE THREAD

**Issue by sundong212** [niklasvh/html2canvas#2761]
Please make sure you are testing with the latest [release of html2canvas](https://github.com/niklasvh/html2canvas/releases). 
Old versions are not supported and issues reported for them will be closed.

# Please follow the general troubleshooting steps first:

- [ yes] You are using the latest [version](https://github.com/niklasvh/html2canvas/releases)
- [ yes] You are testing using the non-minified version of html2canvas and checked any potential issues reported in the console

### Bug reports:

I tried to convert my whole page to canvas using the library, but it rendering blank of the canvas. It's kind of a large canvas around 1200px x 3000px. I am able to see the the log of html2canvas, which is showing it finished rendering.

I'm using gulp to build my page to a single html page that containing HTML, JS, CSS. And I'm not using any of the library like React or Vue. It basically just plain JS.

I browsed for half of the day to try to address the issue, but none of them worked for me, which includeing removing the --apple-system font (I don't have it originally, I'm using PingFang-SC, but changed to Helvetica to reduce the interference)

Also my page contains Chinese letter but I tried to convert only the <p> tag to canvas, and it works, so it shouldn't be the Chinese letter issue.

Moreover, I tried it on many other browsers, including Chrome and Safari on MacOS, Safari on iPadOS (same one having the issue above), iPhone on iOS 14.8 and some Android devcices. All of them are working except IN THE FLUTTER APP WITH flutter_webview, which the core is WKWebview

Basically because of the configuration of the project, all of the image in the page are rendering via base64, which doesn't have any cors issue.

Any help would be appreciated.
Thank you very much!

### Specifications:

 * html2canvas version tested with: 1.3.3
 * Browser & version: flutter_webview, WKWebview
 * Operating system: iPadOS 15.0.2

**Comment by XMYuan** [comment]
Is your font-family Is the 'system-ui'?

**Comment by niklasvh** [comment]
Can you please provide a repro on jsfiddle.net
